contracts/Nexus.sol

```diff
@@ -267,6 +262,7 @@ contract Nexus is INexus, EIP712, BaseAccount, ExecutionHelper, ModuleManager, U
         else if (moduleTypeId == MODULE_TYPE_EXECUTOR) return true;
         else if (moduleTypeId == MODULE_TYPE_FALLBACK) return true;
         else if (moduleTypeId == MODULE_TYPE_HOOK) return true;
+        else if (moduleTypeId == MULTITYPE_MODULE) return true;
```

Comment:
could we rename it MODULE_TYPE_MULTI instead for consistency ?

Reply:
done